Sources/ConfidenceProvider/ConfidenceClient/RemoteConfidenceClient.swift

```diff
@@ -187,6 +187,7 @@ public enum ConfidenceClientCredentials {
 }
 
 public enum ConfidenceRegion: String {
+    case global = "global"
     case europe = "eu"
     case usa = "us"
 }
```

Comment:
do we really need "string backed enums" when we do not use the strings?
```suggestion
public enum ConfidenceRegion {
    case global
    case europe
    case usa
}
```

Reply:
Fair point, I think we can just keep enums! I'll keep it in this PR.